src/applications/vaos/covid-19-vaccine/components/VAFacilityPage/FacilitiesRadioWidget.jsx

```diff
@@ -74,11 +74,13 @@ export default function FacilitiesRadioWidget({
               disabled={loadingEligibility}
             />
             <label htmlFor={`${id}_${facilityPosition}`}>
-              <span className="vads-u-display--block vads-u-font-weight--bold">
-                {name}
-              </span>
-              <span className="vads-u-display--block vads-u-font-size--sm">
-                {address?.city}, <State state={address?.state} />
+              <span className="vads-u-display--block">
+                <FacilityAddress
+                  name={name}
```

Comment:
I think passing the `name` here means it is put in an `h2`, which I don't think we want in a radio button list (unless I missed that in the ticket). So we could leave this prop out and put the name above `FacilityAddress` similar to how it's marked up prior to this change.

Reply:
Good catch. Removed the `name` and `level` props and restored the facility name changes.